modules/javafx.graphics/src/test/java/test/javafx/concurrent/ServiceTestBase.java

```diff
@@ -61,8 +60,11 @@ protected Executor createExecutor() {
         };
     }
 
-    @Before public void setup() {
-        factory = setupServiceFactory();
+    // NOTE: This should be reverted once parametrized class tests are added to JUnit5
+    //       For now, tests call this manually
+    // @BeforeEach
```

Comment:
I wonder if we should log a JBS ticket to fix parameterized tests by reverting `@BeforeEach`, and then simply refer to it?
This way, once junit supports that, we can easily grep the code.

(I've used a different comment for same purpose)

What do you think?

Reply:
As mentioned in another comment, it's unlikely we'll spend more time on this later on. I will remove all the NOTE comments and commented out imports and such.